Formula/adam.rb

```diff
@@ -0,0 +1,37 @@
+class Adam < Formula
+  desc "Genomics analysis platform built on Apache Avro, Apache Spark and Parquet"
+  homepage "https://github.com/bigdatagenomics/adam"
+  url "https://search.maven.org/remotecontent?filepath=org/bdgenomics/adam/adam-distribution-spark2_2.11/0.23.0/adam-distribution-spark2_2.11-0.23.0-bin.tar.gz"
+  sha256 "08e43ab689977787635b1c92b056b387e3d19fd2ae173fbaf5cf3a7c948188a5"
+  # tag "bioinformatics"
```

Comment:
You can remove this line. All formulae in this repo are bioinformatics.
If it's published, please add…
```ruby
# doi "…"
```

Reply:
In URI form? `# doi "http://dx.doi.org/10.1145/2723372.2742787"`

Reply:
Yes, please. Remove the `dx.`

Reply:
And let's use `# cite` rather than `# doi`. Sometimes we reference ArXiV URLs or a web page, which are not a DOI.